omnipod/src/main/java/info/nightscout/androidaps/plugins/pump/omnipod/ui/PodHistoryActivity.java

```diff
@@ -272,6 +273,12 @@ private void setValue(OmnipodHistoryRecord historyEntry, TextView valueView) {
                     }
                     break;
 
+                    case PLAY_TEST_BEEP: {
+                        if (historyEntry.getData() != null) {
+                            valueView.setText(historyEntry.getData());
```

Comment:
I think we forgot to verify whether this works as expected. It should show up in the Pod History (which you can find in the Pod Mgmt menu)

Reply:
![Screenshot_20201121-212937_AndroidAPS](https://user-images.githubusercontent.com/1446257/99887011-2af30c80-2c41-11eb-9313-bb858a769401.jpg)

Reply:
Thanks!